### PR TITLE
chore: Add Quasar starter files

### PR DESCRIPTION
**Pull request by rafaelmaiach**
## Type
Chore

## Added
- Quasar starter files as boilerplate

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "build": "quasar build",
+    "build:pwa": "quasar build -m pwa",
+    "build:mobile:capacitor:android": "quasar build -m capacitor -T android",
+    "build:mobile:capacitor:ios": "quasar build -m capacitor -T ios",
+    "lint": "eslint --ext .js,.vue src",
+    "test": "echo \"No test specified\" && exit 0"
+  },
+  "dependencies": {
+    "@quasar/extras": "1.5.0",
+    "axios": "0.19.2",
```
Comment (oliverstr) [inline package.json]: Acho que não vamos precisar de axios já que vamos usar o sdk do firebase.

File: src/layouts/MyLayout.vue [hunk 2]
```diff
@@ -0,0 +1,133 @@
+<template>
```
Comment (oliverstr) [inline src/layouts/MyLayout.vue:1]: Isso é boilerplate? Talvez a gente possa remover.

File: package.json (hunk 1 above)
Comment (rafaelmaiach) [reply package.json]: Removed on https://github.com/oliverstr/crud-social/pull/1/commits/31b2e001593f12e1b4431d5c77cfb15422f60dd6

File: src/layouts/MyLayout.vue (hunk 2 above)
Comment (rafaelmaiach) [reply src/layouts/MyLayout.vue:1]: Removed on https://github.com/oliverstr/crud-social/pull/1/commits/4545e945df44cef3dbc141817b5a8ec2bc9431e9

**Review by oliverstr — APPROVED** [review]

**Pull request merged**